src/cli.rs

```diff
@@ -475,7 +475,7 @@ fn init_commands<'a, C: CLI<'a>>() -> Vec<Command<'a, C>> {
                         io,
                         "{} {}",
                         UserAddress(pub_key.address()),
-                        wallet.balance(&pub_key.address(), &asset.item).await
+                        wallet.balance_breakdown(&pub_key.address(), &asset.item).await
```

Comment:
At the end of this loop, we should also print something like `Total {}, wallet.balance(&asset.item).await`